src/reducers/index.js

```diff
@@ -25,8 +27,10 @@ export default combineReducers({
   namespaces,
   pipelines,
   pipelineRuns,
+  secrets,
   tasks,
-  taskRuns
+  taskRuns,
+  serviceAccounts
```

Comment:
order alphabetically unless there's a particular need for this reducer to run after others (which should mostly be avoided)